manifests/charts/install-Ambient.md

```diff
@@ -0,0 +1,89 @@
+# Istio Ambient Installation with Helm
+
+This doc provides installation guide of Istio Ambient mesh using Helm.
+
+## Setup Repo Info
+
+```console
+helm repo add istio https://istio-release.storage.googleapis.com/charts
+helm repo update
+```
+
+_See [helm repo](https://helm.sh/docs/helm/helm_repo/) for command documentation._
+
+## Installing the Components
+
+### Installing Base Component
```

Comment:
I know that people have been against a top-level chart in the past, but really I think Ambient, as a more opinionated install, needs a top-level chart that at least collates the non-optional bits.

- cni
- istiod
- ztunnel

the top-level chart doesn't have to do anything but be a values.yaml with named subcharts - I don't think this piecemeal install of non-optional components is very good ambient install UX. We could halve the length of this install with a simple named top-level `ambient` chart that had the requisite subcharts as pure dependencies.

@howardjohn or @linsun or anyone else - thoughts on this? I know we want to avoid a heavy top-level chart, but I think we need a simple one that collates the required components.